packages/@react-spectrum/picker/src/Picker.tsx

```diff
@@ -126,25 +136,14 @@ function Picker<T extends object>(props: SpectrumPickerProps<T>, ref: DOMRef<HTM
 
   // Measure the width of the button to inform the width of the menu (below).
   let [buttonWidth, setButtonWidth] = useState(null);
-  let {scale} = useProvider();
-  useLayoutEffect(() => {
```

Comment:
Does resize observer always get called on the initial render? I think at least when not supported it wouldn't based on my reading of the code.

Reply:
O good point, forgot about the unsupported case. Otherwise yes, it seems to be called on the initial render https://drafts.csswg.org/resize-observer/#ref-for-element%E2%91%A3
```
ResizeObserver’s notifications can be used to respond to changes in Element's size. Some interesting facts about these observations:

Observation will fire when watched Element is inserted/removed from DOM.

Observation will fire when watched Element display gets set to none.

Observations do not fire for non-replaced inline Elements.

Observations will not be triggered by CSS transforms.

Observation will fire when observation starts if Element is being rendered, and Element’s size is not 0,0.
```

Reply:
It might be better to implement that in useResizeObserver's fallback case. Trigger a "resize" event on mount?